PersonApi/readme.md

```diff
@@ -0,0 +1,29 @@
+# Person Api
```

Comment:
This should be the readme in the root of the repo (not a project)

Reply:
All done